app/views/pages/home.html.erb

```diff
@@ -38,6 +38,8 @@
           <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
         </svg>
       </a>
+        
+        <p>You may still be eligible to teach even if you cannot get an adviser. <a href="https://getintoteaching.education.gov.uk/">Find out more about teaching in England</a>.</p>
```

Comment:
```suggestion
        <p>You may still be able to teach even if you are not eligible for an adviser. <a href="https://getintoteaching.education.gov.uk/">Find out more about teaching in England</a>.</p>
```

I wonder whether we should only talk about eligibility in the context of an adviser @EllieNodder, rather than in terms of teaching? 

t's hard because we are trying to cover ITT candidates, returners and international trainees/teachers all in one sentence - even this doesn't cover training explicitly!